rose/client/main.py

```diff
@@ -95,15 +96,27 @@ def load_driver_module(file_path):
 
 def main():
     logging.basicConfig(level=logging.INFO)
-    if len(sys.argv) < 2:
-        log.info('usage: rose-client drive-module')
-        sys.exit(2)
+    parser = argparse.ArgumentParser(description="ROSE Client")
+    parser.add_argument("driver_file", help="The path to the driver file")
+    parser.add_argument("--server-address", dest="server_address",
+                        help="The server FQDN/IP address to connect to."
+                        " If not specified, localhost will be used.")
```

Comment:
Please define the arguments int the same order we expect them:

    --server-address
    driver_file

Regarding server address - we like to have both short and long option. --server-address is very readable but pain to type again and again, -s will be much nicer.

We can do:

    parser.add_argument("--server-address", "-s", dest="server_address", ...)
    
If server address is not specified, we can define a default using default="localhost".

In the help, please do not use terms like FQDN or IP address, instead use the term "server address" and show example of both  "example.com" and "192.168.0.100".

In the driver_file help, write: "The path to the driver python module".

Reply:
done